mir-module/src/main/resources/META-INF/resources/images/svg_icons/download-generic.xml

```diff
@@ -0,0 +1,32 @@
4uUapMuQom8lfPrJpFDatadeo1sGnUpFnBl8uxf38fuVjy3qSiD5ZU3ArpVbxZ5Mi1IKYxHMirQSkRiFdh2gODQLLJMkAnRkIIUWwpI42WKiMJXrAkpWqREhpjhVG0PQnhUoJxSNKQETUdgw6QE0Y5xFgUaaxdRjVGCCS+IMV0s5zlG2UGUGaS/V+sKKVITCKKmmDteliELde7aeTEdMw0IAchXWYD7vpVTQHmOYyrG4ModqFVYjgKH5HDKgeZomafkFHxOXVWGPdy4Wkx2TDcoAKdVucThWLC/L0e2TNBddaKQNDrphZZMBMk8qsMzze/NDVuKxkgpC3j2HzXUR7xlme8xdY4JqKWJi5Ls9LzpiWH83pFzidNWLLgFprYXAV/PsalqzVZ8KJp7WLFa5tkbY9wsWJf+jHdrrnC751IFyVRGSMxGSWaDBDh3FeA5L+lGmnIPjKHSS96cuNOnNeejMRlwiULq8Zp10Xyaj3WrwKJwit/lAfVoDA+1Yt4gNh2yiYrVrsix85IEKxbvBh15UU3GbHX1SKg7zsEWKQsiFRcBBIV7Ff5ivZERVXKXiEue0TUjXJYaBnV4uwYYbFKQtpyMEtbum32OVyMhoQgzNUJSKHrNSnVAU27KqpdWMkOD93ITw8hkCeKNLrbEdDpuqwJZ0jOlNwCcOFPgLOBd9Mt8qBdE96nZ1yRMbyd9vzI2PaV8njtynu4Mt4klGXtlGkrLFFZ3RtYA2Tv/gfNUTONRoJIIhG/zXKYzSJXlWldBLRcK6iEZ+jJccmdAYumpfkGyXNwykVbQ7WLSbVSQ6UKkOj0JSooSaSnpkjjebLPdIEF9Rgxe8mEMuCGkEqbdk6ZMKEMuCGk0qZl51QIE8qAG0IqbVp2TpUwoQy4IaTSpmXn1AgTyoAbQiptWnZOwYQy4IaQSpv2oDsJf/hynhzIhcJpgsOd75e1U4c03246pJOWhWUoefmADEBgb/Z969UXToN1udJ4t8YEwpatQXxsLnBRrVaSprp2si6gZGcd54bGfW9GGl9W7LhnAE7eYkCRCayJNep94jd8ZhuUBKdcZMt0KqfWyHj5UYnhIlUOtbEpJr6lkN8PL4e33EtWRz45PWpa7/EEnXywdV5bxR87xWH/59FSe8aYgZEoESaUATeEVKMxvbvIi6IpilTXq5oP1dKz0OUhYnyvtXXbUxFp1P+ToxdWuj1skPgHYuSaQFe346bjYwRmF2EVGPfgTJhQBtwQUmnTsnM6CBPKgBtCKm1adk6DMBFSadOy565LRhOklFJKKaWU8qj7CBPKgBtCKm1adk6bMKEMuCGk0qZl53QSJpQBN4RU2rTsnC7ChDLghpBKm5ad002YUAbcEFJp07JzeggTyoAbQiptWvZ5LhBhQhlwQ0ilTcvOKRMmh96sXSaLzeHy+AKhSCyRyuQp2iaLzeHy+AKhSCyRyuQpOk0Wm8Pl8QVCkVgilclTdJksNofL4wuEIrFEKpOn6DZZbA6XxxcIRWKJVCZP0WOyuDy+QBgNO7NZVr14NICz166269IJF5qIhDmcg8wVBh5wIKR+RkeZcDNPQWrApNyHx0tNLwkhCPju58/OEoO+dbp/W6rHZzIM8OI9GsaYHUqCvB+3SMptBbD+xWOL5VZ+6egRYh7Pp3BdbKfZRqFaPXIm43oeJcGbJmIWRjXszyN+wjXi7f3rbye1wW53HuaCPwvnhVWniqBkRrDaPb/cngTgd4iP9w0ymWU/d1cbEF2oHIiDVha3h3g/UDzjOEHxLX60PsfA7tAifrP3+H+CkuEkJAAsi0QZUP9nFLxRMQPWOMKi6cuNEm0dOr6GYtFgjSN46mNTcqUPAlMQ3p3JW1YBZ50TfM8MrUWM1byTvfLUw1zIpMW+/lgLaURCLTLjJWZJ8K5HyLTPSIztk2jTz+noGwl2Q+LpHdxlhs2E9VeyqXsymowlg2TSTVok2cdFStLNQnoATh67xcX20G8yf801GO5LYCY9P99JvfRzwKYEo4QbQio9Tvnj0g4AAAAAAAAAAAAAAACCT5HYI21adk4dwoQy4EIqbWbXJUwZN4RU7745orykE80TH8C23JxOHo5ljBtCaTPni+VeJE4CZfxJccvAvJ4ETkf/7aYRJpQBN4RU2szqZEJBMG4kAgN4CjEkCk5Jtjs5pHz/BqmQuPDItBxcouxs4YYPc0jHRzoUyphZahJn3/3tA2j4D4oBQlTAKrACDBQIMicAKqCiAr7mBKDqAuciAKQCAASkBNAIAACuCEQJAASFQpR+/m3wFJg5AqREgEqRwAoIBgMBEAGNCIQAApAywZciVAvxLwAAKgBBQEoAjQAA4IpAlAAggEaHNZAdxDChDLgh5lo2z46yGmttq71qnNkoZMJOYbjlZXMrHj18FNYFSP7dz5O4J7kthlr1DmDOfnCSYd4/ZOs3CdP8wJh8cyU2XDwLjzeyYe3/Q01/AG17D017Ym6j6v5DZRMVlRLllWW1T15TUkZxboQpvxjzahhyEvTFqa6oq7BgQ5uVoslUp0pVWs4uLUWajvtEWRiHkcUljY2QRFjiaAKzyFUYKgSB/KAUNwiu3wbbX2yL6clyDUxHhuOG6mBBs6FaUkxkYkoy64JoTGBkBB73MIBYhOnplaefkHsEcw4Q2dorsrRrpnYOzP7uO6RVR6XmEQJffyCGV+V7D5LAdXbAif3e28EFLBkw5X9E4AQodoRrknB/DNJwEP8RIvwDUsq4kzKE7k7BEH/Og1fh/9Y3Ve2Eflo1rp9DDG51lat9HjgM8xseKzHcmGy16dWKZwBTCGlQ4Z8Za4KwGg0mHThKvDqJ1WIChWF6Q2PEOHJp/ONdHMVfBj0/qnWDNSSta+CXgZ39O35SQfHr0K+uJSP+FL7/8QNmN0f4pSu/CjCzmyETVRPJLH4KzU/MEuO9/wSBYAlzmJNKqC2+hQH+Wt6+d1EMfYGFptWXfBam7EO8AgAAAA==) format('woff2');
```

Comment:
I think it is questionable to include the entire font in each generated SVG image